docroot/modules/custom/sitenow_people/sitenow_people.module

```diff
@@ -699,7 +699,17 @@ function sitenow_people_preprocess_paragraph(&$variables) {
           }
         }
       }
+      break;
 
+    case 'uiowa_contact':
+      $admin_context = \Drupal::service('router.admin_context');
+      if (!$admin_context->isAdminRoute()) {
+        if ($paragraph->hasField('field_uiowa_contact_address')) {
+          if ($paragraph->field_uiowa_contact_address->country_code == 'US') {
+            $variables["content"]["field_uiowa_contact_address"][0]["country"]["#value"] = ' ';
```

Comment:
Can we switch these double quotes to single quotes, please?

Also, if I'm not mistaken, what the logic is doing here is checking the country code of the first entry of the multi-value `field_uiowa_contact_address` (via magic method) and then setting the value of the country field for the first entry to an empty string. It appears that this would not prevent a second entry from showing the country if the country code was `US`.

Reply:
Interestingly, it does seem to prevent multiple entries from hiding the US in various combinations of US and non-US addresses.

![image](https://user-images.githubusercontent.com/49074566/152871617-9c5f43bc-6644-4cfb-a629-fd8160b570cf.png)

Reply:
hmm It works for me when setting first entry = US and second to a different country?

Reply:
Op. Sorry, Cory! That works as desired then! 

> > @cory-skeers ah, yes. It then displays '%placeholder' w/ errors. Hmmm we could add a class that sets the display to none? Not sure if that's ideal - but would be more accurate for screen readers?
> 
> I think that'd be closer to ideal. There's the question why we'd hide something from visual display that would be read for screen readers, but I think the request stems primarily from a visual design, so maybe that's not a concern here.

I can try going about it the class route? @pyrello any objections?

Reply:
why... did it add that comment here 🙃

Reply:
Maybe I'm misreading the code.

Reply:
No, the way you're reading it would make perfect sense... the address field is just incredibly wonky